scraper/src/page-scraper/chunk-scraper/chunk-helpers/GetFaculty.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Extracts the name of the faculty that offers the course that is being parsed
+ * @param faculty: Line that contains data about the faculty field
+ * @returns { string }: The faculty that offers the current course
+ */
+const getFaculty = (faculty: string): string => {
+    // Faculty is a string
+    if (!faculty || faculty === " ") {
+        console.error(new Error(`Invalid Faculty: ${faculty}`));
```

Comment:
``if (!faculty)`` works as well / why would that be a " " char?

Reply:
feel like it may be best to keep it for consistency because getCampusLocation and getSchool also have the same checks, and i don't wanna break anything! But if u want me to remove, then i shall